plan/logical_test.go

```diff
@@ -1,5 +1,8 @@
 package plan_test
 
+// TODO(affo): what to do now that compilation produces a Program?
+//  Should the logical and physical plans be accessible?
+/*
```

Comment:
I don't want to lose these tests. I have an example of how I did similar testing:
https://github.com/influxdata/flux/blob/55fb97f40907408a3e2ccaf2f3d2c0e11a10c2f0/lang/compiler_test.go#L20

You can access the plan in the `lang.Program`'s `Spec` member.  I think you'll want to do something similar here.